test/format/format_config_def.c

```diff
@@ -364,6 +364,10 @@ CONFIG configuration_list[] = {{"assert.read_timestamp", "assert read_timestamp"
   {"transaction.implicit", "implicit, without timestamps, transactions (percentage)", 0, 0, 100,
     100, V_GLOBAL_TRANSACTION_IMPLICIT},
 
+  {"transaction.operation_timeout_ms",
+    "requested limit on the time taken to complete operations in this transaction", 0, 0, 0,
+    UINT_MAX, V_GLOBAL_TRANSACTION_OPERATION_TIMEOUT_MS},
```

Comment:
I think you can specify the default here in the config struct, which should be 2000.

Reply:
Fixed in [4de96ae](https://github.com/wiredtiger/wiredtiger/pull/9357/commits/4de96ae50ca00ba0bde584aabe445769f4f6ac27)

Reply:
Can you confirm that that is being passed into transactions in test format? Like manually debugging or similar.